src/h265_nal_unit_parser.cc

```diff
@@ -74,6 +74,20 @@ H265NalUnitHeaderParser::ParseNalUnitHeader(const uint8_t* data,
   return ParseNalUnitHeader(&bit_buffer);
 }
 
+bool H265NalUnitHeaderParser::ParseNalUnitType(const uint8_t* data, const size_t length,
+      NalUnitType& naluType) noexcept {
+  if (length == 0) {
```

Comment:
why this check? The rest of the code should work with length == 0 (I just added a unittest to make sure)